containers/payments/ExpInput.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Input } from 'react-components';
+import { c } from 'ttag';
+import { isNumber } from 'proton-shared/lib/helpers/validators';
+
+const isValidMonth = (m) => !m || (isNumber(m) && m.length <= 2);
+const isValidYear = (y) => !y || (isNumber(y) && y.length <= 4);
+
+const ExpInput = ({ month, year, onChange, ...rest }) => {
+    const handleChange = ({ target }) => {
+        const [newMonth = '', newYear = ''] = target.value.split('/');
+
+        if (target.value.includes('/')) {
+            onChange({
+                month: isValidMonth(newMonth) ? newMonth : month,
+                year: isValidYear(newYear) ? newYear : year
+            });
+        } else if (newMonth.length > 2) {
+            // User removes the '/'
+            return;
+        } else if (month.length === 2) {
+            // User removes the '/' and year is empty
+            const [first = ''] = newMonth;
+            onChange({
+                year,
+                month: isValidMonth(first) ? first : month
+            });
+        } else {
+            const [first = '', second = ''] = newMonth;
+            onChange({
+                year,
```

Comment:
```suggestion
                year: '',
```

I think we should maybe reset `year` here

That would allow the user to for example select everything and delete the year too